internal/blocksync/pool.go

```diff
@@ -786,8 +786,6 @@ OUTER_LOOP:
 		}
 
 		retryTimer := time.NewTimer(requestRetrySeconds * time.Second)
-		defer retryTimer.Stop()
```

Comment:
please revert this change

Reply:
yeah, I have removed it and added manual retryTimer.Stop()